src/main/java/teammates/common/util/GaeVersionApi.java

```diff
@@ -0,0 +1,80 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.appengine.api.modules.ModulesService;
+import com.google.appengine.api.modules.ModulesServiceFactory;
+
+/**
+ * A wrapper class for GAE application admin API. e.g. version management
+ */
+public class GaeVersionApi {
+    /**
+     * Maximum number of versions to query.
+     * The current value will include the current version and its 5 preceding versions.
+     */
+    private static final int MAX_VERSIONS_TO_QUERY = 6;
+    
+    /**
+     * Default constructor.
+     */
+    public GaeVersionApi() {
+    }
+    
+    /**
+     * Gets all available versions.
+     */
+    public List<Version> getAvailableVersions() {
+        ModulesService modulesService = ModulesServiceFactory.getModulesService();
+        List<String> versionListInString = new ArrayList<String>(modulesService.getVersions(null)); // null == default module
+        List<Version> versionList = new ArrayList<Version>();
+        for (String versionInString : versionListInString) {
+            versionList.add(new Version(versionInString));
+        }
+        Collections.sort(versionList);
+        return versionList;
+    }
+    
+    /**
+     * Gets the current version of the application.
+     */
+    public Version getCurrentVersion() {
+        ModulesService modulesService = ModulesServiceFactory.getModulesService();
+        return new Version(modulesService.getCurrentVersion());
+    }
+    
+    /**
+     * Gets a few recent versions for log query.
+     * @return a list of default versions for query.
+     */
+    public List<String> getDefaultVersionIdsForLogQuery() {
```

Comment:
A method should not be named based on the caller. A method can be called by any other class. Name it based on what the method does.